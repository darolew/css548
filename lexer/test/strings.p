""
''
"hello, world"
'hello, world'
"hello, 'world'"
'hello, "world"'
"hello, 'world"
'hello, "world'
'hello,
world'
"hello,
world"
"(* not a comment *)"
'(* not a comment *)'
"{ not a comment }"
'{ not a comment }'
"while"
"if"
";"
'while'
'if'
';'
