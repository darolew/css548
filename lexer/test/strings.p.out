309   ystring          
309   ystring          
309   ystring          hello, world
309   ystring          hello, world
309   ystring          hello, 'world'
309   ystring          hello, "world"
309   ystring          hello, 'world
309   ystring          hello, "world
309   ystring          hello,
world
309   ystring          hello,
world
309   ystring          (* not a comment *)
309   ystring          (* not a comment *)
309   ystring          { not a comment }
309   ystring          { not a comment }
309   ystring          while
309   ystring          if
309   ystring          ;
309   ystring          while
309   ystring          if
309   ystring          ;
